src/app.ts

```diff
@@ -4,15 +4,29 @@ import cors from 'cors';
 import * as dotenv from 'dotenv'
 import user from './routers/user';
 import group from './routers/group';
+import { errorHandler } from './middlewares/errorHandler';
+import {processListener} from './process'
+import { morganMiddleware } from './middlewares/morgan.middleware';
+import { log } from './middlewares/winstonLogger';
 const app = express();
 dotenv.config()
 
 // Enable Cross Origin Resource Sharing to all origins by default
 app.use(cors());
 // Transforms the raw string of req.body into json
 app.use(express.json());
+//load common middlewares
+app.use(morganMiddleware);
```

Comment:
You'd better mane your middleware by functionality: what it does, not its origin.

Reply:
fixed